gremlin-dotnet/test/Gremlin.Net.IntegrationTest/Driver/GremlinClientTests.cs

```diff
@@ -167,6 +168,25 @@ public class GremlinClientTests
             }
         }
 
+        [Fact]
+        public async Task ShouldReturnResponseAttributes()
+        {
+            var gremlinServer = new GremlinServer(TestHost, TestPort);
+            using (var gremlinClient = new GremlinClient(gremlinServer))
+            {
+                var expectedResult = new List<int> { 1, 2, 3, 4, 5 };
+                var requestMsg = $"{nameof(expectedResult)}";
+                var bindings = new Dictionary<string, object> { { nameof(expectedResult), expectedResult } };
+
+                var response = await gremlinClient.SubmitAsync<int>(requestMsg, bindings);
+
+                var resultSet = response.AsResultSet();
+
+                Assert.NotNull(resultSet.StatusAttributes);
```

Comment:
Any reason to not explicitly assert the "host" attribute as I did in my tests?

https://github.com/apache/tinkerpop/commit/81fd8403b77a440b99a8757d154e52f7aa63e297#diff-224b6c96eaf313833e0c1bbfe94b0b68R104

Reply:
I did check for host because I was not setting that and was not sure if this is returned in all type of test environments. But if this is added in all test environment, I will add that

Reply:
StatusAttributes is a typed Json. I added test for to extract the value and check for host